paasta_tools/chronos_serviceinit.py

```diff
@@ -290,11 +313,26 @@ def status_chronos_jobs(jobs, job_config, verbose):
 def perform_command(command, service, instance, cluster, verbose, soa_dir):
     chronos_config = chronos_tools.load_chronos_config()
     client = chronos_tools.get_chronos_client(chronos_config)
+    job_config = chronos_tools.load_chronos_job_config(
+        service=service,
+        instance=instance,
+        cluster=cluster,
+        soa_dir=soa_dir,
+    )
     complete_job_config = chronos_tools.create_complete_config(service, instance, soa_dir=soa_dir)
     job_id = complete_job_config["name"]
 
     if command == "start":
-        start_chronos_job(service, instance, job_id, client, cluster, complete_job_config, emergency=True)
+        start_chronos_job(
+            service,
+            instance,
+            job_id,
+            client,
+            cluster,
+            job_config,
+            complete_job_config,
+            emergency=True,
+        )
```

Comment:
Can you kwargs this? As well as `stop_chronos_job` and `restart_chronos_job` while you're at it.